services/cd-service/pkg/service/overview.go

```diff
@@ -95,15 +95,21 @@ func (o *OverviewServiceServer) getOverview(
 		for envName, config := range envs {
 			var groupName = mapper.DeriveGroupName(config, envName)
 			var envInGroup = getEnvironmentInGroup(result.EnvironmentGroups, groupName, envName)
+			argocd := &api.EnvironmentConfig_ArgoCD{}
```

Comment:
Is that actually needed?

Reply:
I want to make sure that no application with argocd with `nil` value causes a panic during the transform